custom-words.txt

```diff
@@ -2066,3 +2066,4 @@ regionality
 Resource
 Metastores
 maintenancewindows
+Exprired
```

Comment:
Is this spelling correct?

Reply:
Yes, this is actually coming from another RP that has the incorrectly named property on their end. Since we're just proxying the response, we're having to keep it as-is.

Reply:
This shouldn't have been added to the general dictionary, I opened https://github.com/Azure/azure-rest-api-specs/pull/12579 to move it to an individual override like it was in the previous version (so it would be flagged and fixed in the next version)

Reply:
The right time was to fix it before GA 😉

Reply:
Thank you @nschonni for pointing this out. I wasn't aware there was this provision. I will keep this in mind going forward.